apps/judicial-system/web/src/components/Inputs/InputAdvocate.tsx

```diff
@@ -33,7 +33,6 @@ interface Props {
   name: string | undefined | null
   email: string | undefined | null
   phoneNumber: string | undefined | null
-  onAdvocateNotFound?: (advocateNotFound: boolean) => void
```

Comment:
This component is not only used for defenders. Are we sure we always want to force a lawyer to be selected?